deployment/inventory.yaml

```diff
@@ -8,10 +8,10 @@ all:
           env_name: staging
           k8s_kube_context: microk8s
           k8s_domain_names:
-          - ratom-staging.caktus-built.com
-          k8s_namespace: "ratom-{{ env_name }}"
+          - api-staging.caktus-built.com
```

Comment:
These should all stay as ``ratom``

Reply:
@JeremyGibson I think all of the ``api-`` additions in this file need to revert back to ``ratom-``

Reply:
I left the kubernetes ingress as api